### PR TITLE
UCHV-34 added contact details page

### PR DESCRIPTION
**Pull request by joefitter**
* removed first step bypass
* added contact details step

### CODE REVIEW

**Comment by gxxm** [comment]
:thumbsup: :thumbsup:

File: .gitignore [hunk 1]
```diff
@@ -1,7 +1,5 @@
 .DS_Store
-.npmignore
 apps/*/translations/en
 node_modules
 public
-acceptance_tests/output/
```
Comment (bendiggle) [inline .gitignore]: do we want to remove this?

Comment (joefitter) [inline .gitignore]: maybe... I haven't really looked at it but it doesn't seem that output is currently being output anywhere

Comment (bendiggle) [inline .gitignore]: Yeah come to think of it, since you changed the tests, I haven't seen it anywhere

Comment (joefitter) [inline .gitignore]: let's cal it a "feature".. 😬

**Comment by bendiggle** [comment]
LGTM

**Pull request merged**